src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -165,6 +166,10 @@ private static IOperation CreateInternal(BoundNode boundNode)
                     return CreateBoundLabeledStatementOperation((BoundLabeledStatement)boundNode);
                 case BoundKind.ExpressionStatement:
                     return CreateBoundExpressionStatementOperation((BoundExpressionStatement)boundNode);
+                case BoundKind.InterpolatedString:
+                    return CreateBoundInterpolatedStringExpressionOperation((BoundInterpolatedString)boundNode);
+                case BoundKind.StringInsert:
+                    throw ExceptionUtilities.UnexpectedValue(boundNode.Kind);
```

Comment:
rather than returning OperationKind.None we want to throw? I am fine with throwing, just want to see what is our principle is.